adb-utils.spec

```diff
@@ -1,6 +1,6 @@
 Name:          adb-utils
 Version:       1.5
```

Comment:
We do version bump when we schedule a release and this code is not adding any feature, it's just migrating shell to python script.